src/sampleutil.h

```diff
@@ -43,6 +43,13 @@
 // A group of utilities for working with samples.
 class SampleUtil {
   public:
+    enum CLIPPING {
+        NO_CLIPPING = 0,
+        CLIPPING_LEFT = 1,
+        CLIPPING_RIGHT = 2,
+        CLIPPING_BOTH = 3,
```

Comment:
I do not like CLIPPING_BOTH, using it produces just redundant code. 
In fact we have now a bit filed camouflaged as a enum. 
Using a bitfield struct instead, does not make the code more readable so that enum is ok to me, but 
please use consequent the bit field syntax.
CLIPPING_BOTH is also not that nice because it does not allow future channel count > 2